gulpfile.js

```diff
@@ -55,21 +52,24 @@ gulp.task('favicons:build', ['favicons:clean'], () => {
         .pipe(gulp.dest('./build/assets/favicons'));
 });
 
-gulp.task('favicons:svg', ['favicons:clean'], () => {
+gulp.task('favicons:svg', () => {
     return gulp.src('./assets/images/favicon.svg')
         .pipe(gulp.dest('./build/assets/favicons'));
 });
 
-gulp.task('favicons', ['favicons:build', 'favicons:svg'], () => {
+gulp.task('favicons', gulp.series('favicons:clean','favicons:build', 'favicons:svg', () => {
     return gulp.src('./build/assets/favicons/favicon.ico')
         .pipe(gulp.dest('./web'));
-});
+}));
+
+// clean -> build -> svg
+
 
 gulp.task('images:clean', () => {
     return del(['./build/assets/images/**/*']);
 });
 
-gulp.task('images:banners', ['images:clean'], () => {
+gulp.task('images:banners', () => {
```

Comment:
The intention behind honouring these dependency chains is that these are all individual commands that can be run and replayed in isolation. Did you attempt to preserve all of the dependencies and experience any issues?

Reply:
Introducing dependencies has changed in gulp v4. If you notice I have added `image:clean` task before running [`images` task](https://github.com/elifesciences/journal/pull/1782/files/65815f0f0514c41b212608ccfeacaeb696930497#diff-25789e3ba4c2adf4a68996260eb693a441b4a834c38b76167a120f0b51b969f7R209).

Reply:
We don't use `images:banners` independently in our building process and we only use the `images` task to run all other tasks related to images.

Reply:
> We don't use `images:banners` independently in our building process and we only use the `images` task to run all other tasks related to images.

In development we may use `images:banners` independently if we were adjusting that task and the dependency should be honoured if we do. Or it's removal justified.

Reply:
In this case we should rewrite the whole gulpfile.js because the current implementation doesn't fit with the gulp v4. here is how dependencies work in gulp 4 https://fettblog.eu/gulp-4-parallel-and-series/

Reply:
Ok, I think I get it. Repeating the dependencies could result in a slower execution. I will review accordingly, thanks for the heads up.